make documentation for path (patch, delete, post)

YK9968 opened the pull request.

YK9968 committed:
make documentation for path (patch, delete, post)

RomaShekel committed:
Update openapi.yaml

updare of description

RomaShekel committed:
Update 404.yaml

change a respons of 404 error

RomaShekel commented on swagger/components/responses/404.yaml:
Змінив відповідь 404 помилки на більш загальну, оскільки може стосуватися як води, так і юзерів

RomaShekel committed:
Update water.yaml

Додав час створення, оскільки ми зберігаємо його у форматі Unix, то додавання за замовчуванням базою даних нам не підходить

RomaShekel committed:
Update post.yaml

change createdAt and updatedAt format on unix-format

RomaShekel committed:
Update water.yaml

add updatedAt field in unix-format time

RomaShekel committed:
Update openapi.yaml

RomaShekel reviewed (approved):
Приймаю документацію, тільки змінив опис схеми запису води, оскільки ми зберігаємо час створення і оновлення в unix форматі, то додавання часу бд за замовчуванням нам не підходить. Решта коду в порядку, хороша робота
